model/src/commonMain/kotlin/io/github/droidkaigi/confsched2020/model/Level.kt

```diff
@@ -0,0 +1,14 @@
+package io.github.droidkaigi.confsched2020.model
+
+enum class Level(val rawValue: LocaledString) {
+    BEGINNER(LocaledString("初級", "Beginner")),
```

Comment:
I think it is better to have a string such as "BEGINNER" as a property. `Level(val id :String, val rawValue: LocaledString) ` This is because this class is caused by the change of API response. The property name can be id. 👀 
For example, you have created a class called Level, but you cannot change it that way.

Reply:
I see.
I agree with you. I will fix it!